.eslintrc.yml

```diff
@@ -0,0 +1,31 @@
+extends: "airbnb"
+parser: "babel-eslint"
+parserOptions:
+  ecmaVersion: 6
+plugins:
+  - "react"
+rules:
+  max-len: ["error", {code: 100, tabWidth: 2, ignoreUrls: true}]
+
+  # multi spaces only allowed for aligning variable/import declarations
+  no-multi-spaces:
+    - 2
+    - exceptions:
+        VariableDeclarator: true
+        ImportDeclaration: true
+
+  # we currently use null in many places, so allow == for null checks
+  eqeqeq:
+    - 2
+    - "smart"
+
+  quotes: [2, "double", "avoid-escape"]
+  object-curly-spacing:
+    - 2
+    - "never"
+  new-cap:
+    - 2
+    - newIsCapExceptions: ["kayvee.logger"]
```

Comment:
thanks! i'll remove the rule since we're not using kayvee in this quite yet and it should become a non-issue.